### PR TITLE
Create accessibility-supported.md

### PR DESCRIPTION
**Pull request by dylanb**
Describe how we make decisions on rules and how this impacts ARIA

### CODE REVIEW

File: doc/rule-proposal.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ Before you start coding a new rule for axe-core, you *must* create a Github issu
 
 In addition to giving the axe-core development team an opportunity to provide feedback on the proposed rule, the Github Issue will serve as documentation of that rule for the future.
 
+## WCAG interpretation
+
+Please read our [principles for deciding on rules](./axxessibility-supported.md).
```
Comment (WilcoFiers) [inline doc/rule-proposal.md]: Typo, `axxessibility`.

File: doc/accessibility-supported.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+1. If the feature's use is supported by all platforms: allow, else
+1. If the feature's use does not have a negative impact on accessibility: allow, else
+1. disallow the feature's use until it is supported
+
+In addition, we disallow invalid attributes starting with `aria-` and invalid attribute values as these are highly likely to have a negative impact on accessibility because of the scenarios under which these are likely to occur (this is so we can act as a useful linting tool for developers).
+
+## Best practices
+
+One of the frustrating things about WCAG 2 is that it is noth possible to conform with WCAG 2 and not be accessible and also to be completely accessible and not conform with WCAG 2. We also recognize that there are best practices that significantly improve the usability of application, even though they are not strictly required in order to conform with WCAG 2. We develop the best practice rules to help content developers to identify these and adhere to them.
```
Comment (WilcoFiers) [inline doc/accessibility-supported.md]: Typo `noth`. I think this is a little harsher than it should be. We can just acknowledge that there are techniques you can use that benefit people with disabilities, that aren't required by WCAG.

File: doc/accessibility-supported.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+We recognize that this topic is somewhat controvertial and the rules we have represent Deque's opinion on what constitutes a best practice. We welcome debate and data to help us improve our understanding of this topic and the rules that are encoded in axe-core.
```
Comment (WilcoFiers) [inline doc/accessibility-supported.md]: I don't really think we should be too open to debate. If there's a decent argument to be made for something not to be a best practice, than we shouldn't have had it as a best practice in the first place IMO.

File: doc/accessibility-supported.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# How we make decisions on rules
+
+In order to adhere to the manifesto and at the same time be useful to developers and content creators, we evaluate all rules according to strict normative WCAG 2 interpretation. We also pay very strong attention to the normative portion of WCAG 2 known as [accessibility supported](https://www.w3.org/TR/WCAG20/#accessibility-supporteddef).
+
+## Accessibility supported
+
+Boiled-down, accessibility supported means that in order for a technique to be valid, it must work on all viable platforms for all assitive technology that are widely used and freely available (paraphrased).
```
[the remark this answers is not in the document]
Comment (dylanb) [reply doc/accessibility-supported.md]: This section is just meant to paraphrase the accessibility supported definition from the spec.

I agree with the addition of the ATs we currently test

File: doc/accessibility-supported.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+We currently test the following AT combinations for support
+
+1. VoiceOver and Safari on OS X
+1. VoiceOver and Safari on iOS
+1. JAWS and IE on Windows
+1. NVDA and Firefox on Windows
+1. Talkback and Chrome on Android
+1. Dragon and Firefox on Windows
```
Comment (marcysutton) [inline doc/accessibility-supported.md]: We need some devices to actually support this policy. I personally don't have an Android device or Dragon.

Comment (WilcoFiers) [reply doc/accessibility-supported.md]: I don't think we've been very consistent about testing with Talkback, VO on iOS or Dragon. @dylanb Are you proposing we change this, or is this more of a nice to have list?

Comment (dylanb) [reply doc/accessibility-supported.md]: This is the list we should be testing...

@marcysutton submit a request to get what you need to do your testing...also - Matt I. has access to devices in the Ann Arbor office and the teama11y mailing list is a resource you can also use

**Comment by dylanb** [comment]
@WilcoFiers are the current changes good to go?

**Pull request merged**